modules/ossm-installing-gateway-using-gateway-injection.adoc

```diff
@@ -0,0 +1,246 @@
+:_mod-docs-content-type: Concept
+[id="ossm-about-gateway-injection_{context}"]
+= Installing a gateway by using gateway injection
+:context: ossm-installing-gateway-using-gateway-injection
+
+This procedure explains how to install a gateway by using gateway injection.
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator.
+* You have created an {istio} control plane.
+* You have created an `IstioCNI` resource.
+
+.Procedure
+
+. Create a namespace that you will use to install the gateway.
++
+[source,terminal]
+----
+$ oc create namespace <gateway_namespace>
+----
++
+[NOTE]
+====
+Deploy the gateways associated with the service mesh in a different namespace than the namespace you use to install the {istio} control plane.
```

Comment:
I think we should state that it is a best practice to install the gateway in a different namespace to the control plane namespace. It's still technically possible for users to create gateways in the control plane namespace (which is what OSSM 2 does by default).

In the suggested change I've also attempted to fit in a couple of sentences that describe two common gateway deployment topologies. Although this is brief, I think it would be helpful for users.

```suggestion
It is best practice to install the gateway in a different namespace to the one in which the {istio} control plane has been installed.

The gateway may be installed in a dedicated gateway namespace, where it can be shared by many applications operating in different namespaces. Alternatively, the gateway may be installed in an application namespace, where it serves as a dedicated gateway for the application in that namespace.
```

Reply:
@eoinfennessy  we are highly discouraged from specifically using the phrases "best-practice" or "Red Hat recommends" in the product doc. I doubt it would get through the editorial review  process.

Reply:
@rh-tokeefe, that makes sense. I think it's okay to be prescriptive here. Do you think we could also include the additional topology info? I'm also suggesting a change to the sentence you have here to simplify it and keep the language consistent with the name and aim of the procedure.

```suggestion
[You should] Install the gateway in a different namespace to the namespace the {istio} control plane has been [installed / deployed] in.

The gateway may be installed in a dedicated gateway namespace, where it can be shared by many applications operating in different namespaces. Alternatively, the gateway may be installed in an application namespace, where it serves as a dedicated gateway for the application in that namespace.
```